Sources/PlaybackSDK/Player Plugin/BitMovinPlugin/BitmovinPlayerPlugin.swift

```diff
@@ -7,26 +7,39 @@
 #if !os(macOS)
 import BitmovinPlayer
 import SwiftUI
+import Combine
 
-public class BitmovinPlayerPlugin: VideoPlayerPlugin {
+public class BitmovinPlayerPlugin: VideoPlayerPlugin, ObservableObject {
 
     private let playerConfig: PlayerConfig
-    private weak var player: Player?
+    private weak var player: Player? {
+        didSet {
+            if self.player != nil {
+                listenPlayerEvents()
+            }
+        }
+    }
+    private var cancellables = Set<AnyCancellable>()
 
     // Required properties
     public let name: String
     public let version: String
     
+    public let event: AnyPublisher<Any, Never>
+    private let subject = PassthroughSubject<Any, Never>()
+    
     public init() {
         let playerConfig = PlayerConfig()
         
         playerConfig.playbackConfig.isAutoplayEnabled = true
         playerConfig.playbackConfig.isBackgroundPlaybackEnabled = true
         
-        playerConfig.key = PlayBackSDKManager.shared.bitmovinLicense
+        playerConfig.key = PlaybackSDKManager.shared.bitmovinLicense
         self.playerConfig = playerConfig
         self.name = "BitmovinPlayerPlugin"
-        self.version = "1.0.1"
+        self.version = "1.0.1" // TODO: Get the version from Bundle
```

Comment:
Actually this is the version of the Bitmovin plugin and it could be different between the SDK version. This should be just for internal reference but let's keep it updated and alligned with the SDK version for now.